flink-python/src/main/java/org/apache/flink/python/util/RunnerEnvUtil.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.python.util;
+
+import org.apache.flink.configuration.ConfigConstants;
+import org.apache.flink.util.IOUtils;
+
+import java.io.BufferedOutputStream;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Utils for building the most basic environment for running python udf workers.
+ * The basic environment does not include python part of Apache Beam.
+ * Users need to prepare it themselves.
+ */
+public class RunnerEnvUtil {
+
+	public static final String[] PYTHON_BASIC_DEPENDENCIES = new String[] {
+		"pyflink.zip",
+		"py4j-0.10.8.1-src.zip",
+		"cloudpickle-1.2.2-src.zip",
+		"pyflink-udf-runner.sh"
+	};
+
+	public static List<File> extractBasicDependenciesFromResource(
```

Comment:
Be careful with the format. You can take a look at the methods in `KeyedStream`. 

```
	public static List<File> extractBasicDependenciesFromResource(
			String tmpdir,
			ClassLoader classLoader,
			String prefix) throws IOException {
```

Also check other places.